Week5/Week5.py

```diff
@@ -0,0 +1,31 @@
+# Write a python program create csv file student.csv(sid,sname,city,contact) using 10 records 
+##(in which 5 records input directly and 5 records take input from user). write records into this file.
+
+import csv
+p=open('E:\\22BCA245\\student3.csv','w',newline='')
+w=csv.writer(p)
+header=['sid','sname','city','contact']
+w.writerow(header)
+
+#five records input directly.
+rows=[[1,'OM','Surat',7835657845],[2,'Sai','Rajasthan',9456745612],[3,'Ram','Mumbai',7465645345],[4,'Amit','Chalthan',5675548454],[5,'Radha','Mp',5464384545]]
+w.writerows(rows)
+
+#five records taken input from user. 
+l=[]
+for i in range(5):
+    k=int(input("Enter student id:"))
+    i=input("Enter student name:")
+    n=input("Enter city:")
+    g=int(input("Enter contact number:"))
+    list=[k,i,n,g]
+    l.append(list)

```

Comment:
Can I use extend method instead of append if your answer is no then what is the reason to used append or not to be used extend please explain